data/korath/korath missions.txt

```diff
@@ -257,3 +257,265 @@ mission "Small Scale Delivery"
+				goto title
+
+			label pay
+			`	A ripple of what appears to be dissatisfaction passes through the group, but their leader silences them with a gesture.`
+
+			label title
+			`	The leader turns and speaks to you directly. His eyes never waver from yours even as you wait for the translation. "For many ship skills, parent and child work to learn. For some families, three generations are lost. We will not try to salvage the body box of our kin, for none can bear to fly it. We offer you title to salvage the <EDC-ship> and carry our grief far from here."`
+				accept
+
+	on stopover
+		conversation
+			`You enter the <EDC-ship> wearing environment suits. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them.`
+				to display
+					"flagship crew" > 1
+			`You enter the <EDC-ship> wearing an environment suit. It's been several days since the crash and Korath ships run hot. This will be an unpleasant task in the suits and perhaps impossible without them, especially with no crew helping you.`
+				to display
+					"flagship crew" <= 1
+			`	While your engineer works on refreshing the atmosphere with outside air, the rest of the crew gathers the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+				to display
+					"flagship crew" > 2
+			`	While your crew works on refreshing the atmosphere with outside air, you gather the bodies and places them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest.`
+				to display
+					"flagship crew" == 2
+			`	One by one, you gather the bodies and place them in the fibrous, vaguely egg-shaped coffins the Korath provided. As you move them to your cargo bay, you find they stack together like a clutch of eggs in a nest. It takes a while to refresh the atmosphere with outside air before the smell is tolerable.`
+				to display
+					"flagship crew" < 2
+			`	You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.`
+			`	The crew makes a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
+				to display
+					"flagship crew" > 1
+			`	As a precaution, you make a last sweep to move debris, both to assess for repairs later and to make sure you haven't missed anyone, or parts of them. You clean and disinfect as best you can and return to your ship for the trip to <destination>.`
```

Comment:
Is it intentional that `You return to the Korath ship. In the living quarters you carefully gather all the personal effects you can find and box them for transport. You've no way of knowing if those in the photos are now safely recovering or lying in your hold.` is not included in this line?

Reply:
Why would it be on that line? Are you asking to merge the paragraphs?

Reply:
Sorry, I misunderstood what the syntax was doing here.